packages/relay/src/lib/subscriptionController.ts

```diff
@@ -0,0 +1,134 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// import { Poller } from './poller';
+import { Logger } from 'pino';
+import LRU from "lru-cache";
+import constants from "./constants";
+import crypto from "crypto";
+
+export interface Subscriber {
+    connection: any,
+    subscriptionId: string
+}
+
+const LOGGER_PREFIX = 'Subscriptions:';
```

Comment:
nit: I think we need to add a requestId, like the rest of the relay. But this might be done at a later stage ?